CSharp/BatchExplorer/readme.txt

```diff
@@ -0,0 +1,11 @@
+﻿Once the app is running, you need to configure it with data from your Azure subscription.
+
+1. On the Accounts menu click "Add Default Account Manager". The "Add Account" dialog opens.
+2. In the Account Alias box, type in any name you like. Use the name to create distinct batch jobs.
+3. For the Batch Service Url, go to your subscription and copy the URL. Paste it into the box.
+4. For the Account box, go to your subscription and copy the name of the Batch. Paste it into the box.
+5. For the Key box, go to your suscription and copy the Primary Key, then paste it into the box.
+
+The values are saved automatically, and loaded when you run the file again.
```

Comment:
instead: "After you press ok, the values are saved and can be re-loaded whenever you run BatchExplorer"